manifests/state-nic-configuration-operator/005-configuration.net.nvidia.com_nicconfigurationtemplates.yaml

```diff
@@ -0,0 +1,189 @@
+# 2025 NVIDIA CORPORATION & AFFILIATES
```

Comment:
Please, change ordering numbers in state names

Reply:
Changed the ordering number so that the number do not repeat